Source/WebCore/style/RuleSetBuilder.h

```diff
@@ -34,7 +34,7 @@ class RuleSetBuilder {
     ~RuleSetBuilder();
 
     void addRulesFromSheet(const StyleSheetContents&, const MQ::MediaQueryList& sheetQuery = { });
-    void addStyleRule(const StyleRule&);
+    void addStyleRule(const StyleRule&, const CSSSelectorList* = nullptr);
```

Comment:
Would be good to have a name here since it is non-obvious what the selector list is for.